test/cpp/qps/json_run_localhost.cc

```diff
@@ -42,6 +46,10 @@
 #include "test/cpp/util/subprocess.h"
 
 using grpc::SubProcess;
+typedef std::unique_ptr<SubProcess> SubProcessPtr;
+SubProcessPtr g_driver;
```

Comment:
The Google C++ style guide prohibits global variables of non-POD type:

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables